bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Accordion.java

```diff
@@ -26,22 +26,40 @@
 public interface Accordion extends Container {
 
     /**
-     * Returns the expanded item.
+     * Name of the configuration policy property that stores the default value for the accordion heading's HTML element.
      *
-     * @return The expanded item
+     * @see #getHeadingElement()
      * @since com.adobe.cq.wcm.core.components.models 12.8.0
      */
-    default String getExpandedItem() {
+    String PN_DESIGN_DEFAULT_HEADING_ELEMENT = "headingElement";
```

Comment:
I'd rename it `PN_HEADING_ELEMENT` to map to the JCR property.

Reply:
Yep, but probably `PN_DESIGN_HEADING_ELEMENT` as it's coming from the policy and we use this naming elsewhere.